mathicsscript/__init__.py

```diff
@@ -17,3 +21,6 @@ def load_default_settings_files(definitions):
 
 
 __all__ = ["__version__", "load_default_settings_files"]
+
+if not find_library("libcairo-2"):
+    fix_cairo()
```

Comment:
This kind of thing is better done in setup.py rather than as part of the code.

Reply:
I'll move the GTK+ installation part to setup.py when I have time. As for ``set_dll_search_path()``, it should get run on every invocation of mathicsscript unless cairocffi itself is fixed. I've put in a PR to cairocffi and if they approve it, everythin can be done in setup.py.

Reply:
> I've put in a PR to cairocffi and if they approve it, everything can be done in setup.py.

Awesome!